doc/examples/features_detection/plot_haar.py

```diff
@@ -0,0 +1,70 @@
+"""
+============================
+Haar-like feature descriptor
+============================
+
+Haar-like feature descriptors are a set of simple features which was used in
+the first real-time face detector [1]_. These features can be efficiently
+computed on any scale in constant time, using an integral image [1]_ and by
+subtracting rectangle area. The following example will show the mechanism to
+build this family of descriptors.
+
+References
+----------
+
+.. [1] Viola, Paul, and Michael J. Jones. "Robust real-time face
+       detection." International journal of computer vision 57.2
+       (2004): 137-154.
+       http://www.merl.com/publications/docs/TR2004-043.pdf
+       DOI: 10.1109/CVPR.2001.990517
+
+"""
+from __future__ import print_function
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from skimage.feature import haar_like_feature_coord
+from skimage.feature import draw_haar_like_feature
+
+print(__doc__)
+
+###############################################################################
+# The different Haar-like feature descriptors
+###############################################################################
+# The Haar-like feature descriptors come into 5 different types as illustrated
+# in the figure below. The value of the descriptor consists in the subtraction
```

Comment:
The value of the descriptor is equal to the difference of sums of
intensity values between the green and red rectangles.